frontend/src/components/teamsAndOrgs/orgUsageLevel.js

```diff
@@ -94,8 +96,24 @@ export function OrganisationUsageLevel({ completedActions, orgName, type, userIs
                   }
                   className="tc"
                   value={
-                    // eslint-disable-next-line
-                    <FormattedNumber value={levelPrediction.fee} style="currency" currency="USD" />
+                    <FormattedMessage
+                      {...messages.estimatedTierCostLabel}
+                      values={{
+                        cost: (
+                          <FormattedNumber
+                            value={levelPrediction.fee}
+                            // eslint-disable-next-line
+                            style="currency"
+                            currency="USD"
+                          />
+                        ),
+                        discountLabel: showDiscountLabel ? (
+                          <FormattedMessage {...messages.discountedLabel} />
```

Comment:
I prefer to add `<FormattedMessage {...messages.discountedLabel} />` under a `<span className="f4">`